Water_SCF-vs-Radius/Dimer_scf_vs_radius.py

```diff
@@ -0,0 +1,36 @@
+from wrap_pyscf import *
+
+import pickle
+
+scf_guess = {}
+
+filename = "Dimer_scf-vs-radius-record.pickle"
+
+funct_grps = [[1,0,2], [3,4,5]]
```

Comment:
Writing this was quicker for me, but we can do the "for" loop you suggested.

Reply:
It may have been quicker, but it's harder to maintain. In particular, now that you're going to have to go back and also grab the energies you have to modify 8 lines of code instead of 1. In this particular case it's not so bad because all 8 lines are in one place, but in a more general code base it's easy to miss occurrences.